modular_citadel/code/datums/status_effects/chems.dm

```diff
@@ -72,6 +72,13 @@
 
 //Preamble
 
+/mob/living/verb/toggle_hypno()
+	set category = "IC"
+	set name = "Toggle Lewd Hypno"
+	set desc = "Allows you to toggle if you'd like lewd flavour messages for hypno features, such as MKUltra."
+	client.prefs.cit_toggles ^= HYPNO
+	to_chat(usr, "You [((client.prefs.cit_toggles & HYPNO) ?"will":"no longer")] receive lewd flavour messages for hypno.")
```

Comment:
```suggestion
	to_chat(usr, "You will [((client.prefs.cit_toggles & HYPNO) ? "" : "no longer")] receive lewd flavour messages for hypno.")
```

Reply:
this will give `You will receive lewd ...` and `You will no longer receive lewd ...` instead of
`You will receive lewd flavor` and `You no longer receive lewd`